packages/svelte/svelte-html.d.ts

```diff
@@ -0,0 +1,250 @@
+/// <reference lib="dom" />
```

Comment:
We should explain why we're not exporting it for future readers
```suggestion
/// <reference lib="dom" />
// This file is deliberately not exposed through the exports map.
// It's meant to be loaded directly by the Svelte language server
```